src/components/TableList/TableList.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import MainTable from "../MainTable/MainTable";
+
+function generateTableHeaders() {
```

Comment:
If there is no way to generate these table headers, can you pass them as a prop? Hard coding them makes this component unusable for the Model Detail view.